.circleci/config.yml

```diff
@@ -809,3 +811,42 @@ jobs:
             docker load < docker-image.tar
             docker tag quay.io/influxdb/oss-acceptance:${CIRCLE_SHA1} quay.io/influxdb/oss-acceptance:latest
             docker push quay.io/influxdb/oss-acceptance:latest
+  changelog:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - last-commit
+      - install_core_deps
+      - run:
+          name: Run script
+          command: |
+            set -x
+            git clone --depth=1 https://github.com/orhun/git-cliff
```

Comment:
Should we pin to a specific release version rather than building the latest every time? Might be able to use the docker image instead of installing rust and building it.